src/js/components/Page.d.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, ReactElement } from "react";
+
+interface PageHeaderProps {
+  actions?: [];
+  addButton?: Array<{}> | {};
+  breadCrumbs?: React.ReactNode;
+  supplementalContent?: React.ReactNode;
+  tabs?: [];
+  disabledActions?: boolean;
+}
+
+interface PageProps {
+  className?: [] | {} | string;
```

Comment:
I see in the original propTypes from `src/js/components/Page.js` that `className` can be an object, but when does that actually happen?

Reply:
I have no idea I just copied the props basically. I will look it up later.